pkg/sdk/testint/schemas_integration_test.go

```diff
@@ -13,11 +15,22 @@ func TestInt_SchemasCreate(t *testing.T) {
 	client := testClient(t)
 	ctx := testContext(t)
 
-	// new schema created on purpose
-	schema, cleanupSchema := testClientHelper().Schema.CreateSchema(t)
-	t.Cleanup(cleanupSchema)
+	t.Run("minimal", func(t *testing.T) {
```

Comment:
nit: we had earlier the arrangement, that we put all the integration tests in one function instead of TestCreate, TestAlter etc

Reply:
Changed.